beebop/app.py

```diff
@@ -48,6 +54,31 @@ def check_connection(redis):
         abort(500, description="Redis not found")
 
 
+def generateZip(path_folder):
```

Comment:
Tiny thing, but can you be consistent here with casing (this would be generate_zip following the prevailing style)